src/translators/sql_translator.py

```diff
@@ -1158,59 +1158,48 @@ def query_ids(self,
             for tn in table_names:
                 if "." in tn:
                     table_names.remove(tn)
-
         limit = min(10000, limit)
         offset = max(0, offset)
         len_tn = 0
         result = []
-        stmt = ''
         if len(table_names) > 0:
             for tn in sorted(table_names):
                 len_tn += 1
                 if len_tn != len(table_names):
-                    stmt += "select " \
-                            "entity_id, " \
-                            "entity_type, " \
-                            "max(time_index) as time_index " \
+                    op = "select * " \
```

Comment:
the original query composed results from different tables, this is not the case anymore.

Reply:
> the original query composed results from different tables, this is not the case anymore.

@chicco785 , Yes .. modified as per this [issue](https://github.com/orchestracities/ngsi-timeseries-api/issues/478#issuecomment-874471676)

Reply:
that comments is about enabling the functionality for single entity, not about giving up the listing of entities as it is now.

Reply:
also the missing entitytype for the other queries, was already solved by #486